Manage Website: Links to website

Peteys93 opened the pull request:
Add clickable urls for each website as requested in #10066.

Peteys93 committed:
Manage Website: Links to website

Add clickable urls for each website as requested in #10066.

tsteur commented on plugins/SitesManager/templates/sites-list/site-fields.html:
```diff
@@ -31,7 +31,9 @@
                 <li>
                     <span class="title">{{ 'SitesManager_Urls'|translate }}</span>:
-                    {{ site.alias_urls.join(', ') }}
+                    <div = ng-repeat = "url in site.alias_urls">
```
There is an unneeded `=` sign. Also can you maybe remove the whitespace before and after the `=` here and below after `href`?

tsteur replied:
Some more points: 
* It does not show the comma between urls anymore.
* Can you add a target=_blank so the links open in a new window?
* Also add `rel=noreferrer` to respect privacy and not expose Piwik URL
* Instead of `href` use `ng-href` so link will be only working once actually angular is rendered

tsteur commented:
thanks for that 👍  I need to test it and check values are escaped and formatted correctly. Hope to find some time later. Apart from this I left a tiny note re whitespace / `=` sign.

Peteys93 committed:
Remove extra = and whitespace

Updated style

Peteys93 committed:
New tab, rel noreferrer, ng-href, commas

Changed the urls to open in new tabs, rel=noreferrer, ng-href, and add commas back into the list as requested

Peteys93 committed:
Fix pasting error

Edit out extra </a> and {{ urls}}

tsteur commented:
Sorry for not getting back to you earlier. The change looks good but noticed the URLs are now in a new line in the UI tests: http://builds-artifacts.piwik.org/piwik/piwik/master/18852/SitesManager_loaded.png

I have not tested the latest change. Can you confirm the URL is actually in the same line as the label URLs? (when there are many URLs it is ok when goes over multiple lines)

Peteys93 committed:
Change div to span

Change div to span to have url on the correct line

tsteur commented:
Thank you!
